### ISSUE TITLE
Lasso selection flag

### ISSUE THREAD

**Issue by plankter** [flekschas/regl-scatterplot#35]
It would be really helpful to implement activation of selection via flag or setting in addition to selection via keeping Shift pressed. This will allow users to switch between panning/selection modes via UI switch. Sometimes it is not obvious that Shift has to be pressed.

Best regards,
Anton

**Comment by flekschas** [comment]
Thanks for the suggestion. I've thought about that too and it will definitely come. I just can't give you a time line.

I am happy to integrate a PR if you want to take a stab at it. Should be fairly simple, we just have to refactor `mouseDownShift` such that it is settable via a property (e.g., maybe we call that prob `mouseMode`).

**Comment by plankter** [comment]
I implemented `mouseSelectionMode` boolean property, but unfortunately I could not create a pull request as tests are failing on master branch even without any modifications:

```
./tests/index.js → stdout...
(!) Circular dependencies
node_modules/d3-interpolate/src/value.js -> node_modules/d3-interpolate/src/array.js -> node_modules/d3-interpolate/src/value.js
node_modules/d3-interpolate/src/value.js -> node_modules/d3-interpolate/src/object.js -> node_modules/d3-interpolate/src/value.js

  createRegl()

    Bail out! Unhandled error.

```

**Comment by flekschas** [comment]
Implemented in v0.15 via setting `scatterplot.set({ mouseMode: 'lasso' })`

**Comment by plankter** [comment]
Works great, thank you!
